app/controllers/admin/orders_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Admin::OrdersController < Admin::ApplicationController
+
+	def index
+		if params[:val] == "oldest"
+    		val = "oldest"
+		else
+			val = "newest"
+		end
+
+		if params[:type].present?
+			@orders = Order.status(params[:type].to_i).order_by_time(val).paginate(:page => params[:page], :per_page => 15)	
```

Comment:
```suggestion
			@orders = Order.status(params[:type].to_i).order_by_time(val).paginate(page: params[:page], per_page: 15)	
```

số 15 cho vào setting nhé